js/base/errors.js

```diff
@@ -20,6 +20,7 @@ module.exports = subclass (
                     'NullResponse': {},
                 },
                 'InsufficientFunds': {},
+                'AccountFrozen': {},
```

Comment:
Hey, thanks for your participation! ) One question, though: why wouldn't you use the PermissionDenied or some other AuthenticationError subclass? I wouldn't really want to add a Poloniex-specific exception unless we find more exchanges doing the same with user accounts...

Reply:
I thought this one deserved some additional attention because you do have authorization and its not an authentication issue. Instead you still have access to your historical data, you simply can trade or withdrawal. Ran into this error when accessing a Poloniex account and thought it was a difficult error to replicate.